src/appmixer/pipedrive/crm/UpdateActivity/component.json

```diff
@@ -15,7 +15,7 @@
                     },
                     "done": {
                         "type": "number",
```

Comment:
I thin this should be `string` (same as the CreateActivity), the validation will fail I'd say